aliases/available/xlicp.aliases.bash

```diff
@@ -0,0 +1,8 @@
+cite 'about-alias'
+about-alias 'xclip command shorthands'
+
+if command -v xclip & >/dev/null
+then
+    alias cbci='xclip -i -rmlastnl -selection clipboard' # copy from stdin
+    alias cbcd='cbci -display'                           # copy from particular display
```

Comment:
So would the full description of the `cbcd` alias be something like: `"copy from stdin on a particular display"`?
if so:
* Update the comment - The comment should read as a complete document for the action.
 NOTE: You don't have to use my exact comment above, adjust as appropriate.
* Seems this would be better named as `cbcid` since its an extension of the `*i` alias and would better capture the `stdin` part of the action.

Reply:
I'm hung up on the naming too. I use `xclip` every day. I'm confused about choosing `cb` as the prefix.

If the plugin is about xclip, why not start with `x` something?

MacOS has `pbcopy` and `pbpaste` ... if using `cb`, maybe make aliases that abstract away the OS?

Reply:
I am using clipboard.alias.bash file for this and deleting this one